pkg/cmd/cli/describe/printer.go

```diff
@@ -80,7 +82,41 @@ func NewHumanReadablePrinter(noHeaders bool) *kctl.HumanReadablePrinter {
 	return p
 }
 
-func printParameters(t *templateapi.Template, w io.Writer) error {
+const templateDescriptionLen = 80
+
+func printTemplate(t *templateapi.Template, w io.Writer) error {
+	description := ""
+	if t.Annotations != nil {
+		description = t.Annotations["description"]
+	}
+	if len(description) > templateDescriptionLen {
+		description = strings.TrimSpace(description[:templateDescriptionLen-3]) + "..."
+	}
+	empty, generated, total := 0, 0, len(t.Parameters)
+	for _, p := range t.Parameters {
+		if len(p.Value) > 0 {
+			continue
+		}
+		if len(p.Generate) > 0 {
+			generated++
+			continue
+		}
+		empty++
+	}
+	params := ""
+	switch {
+	case empty > 0:
+		params = fmt.Sprintf("%d blank", empty)
+	case generated > 0:
+		params = fmt.Sprintf("%d generated", generated)
+	default:
+		params = fmt.Sprintf("%d set", total)
```

Comment:
It's what's important to a user - I'm not particularly wedded to particular wording, other than the number of parameters isn't very useful on its own.  "blank" are things the user has to react to, and *should* be providing via process.  "generated" are things they might care about storing for later, but don't need to act on.  "set" is everything else (maybe it should be zero set, or given a different name).